{
  try {
    assertEvent(predicate,0,TimeUnit.MILLISECONDS);
  }
 catch (  TimeoutException e) {
    throw new Error("This should never happen");
  }
}
